fastlane/README.md

```
@@ -0,0 +1,40 @@
+fastlane documentation
+----
+
+# Installation
```

markdownlint-cli2 (0.21.0)

[warning] Heading style
Expected: setext; Actual: atx

(MD003, heading-style)